google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPoolOptions.java

```diff
@@ -50,6 +50,9 @@ public class SessionPoolOptions {
   private final ActionOnSessionNotFound actionOnSessionNotFound;
   private final ActionOnSessionLeak actionOnSessionLeak;
   private final boolean trackStackTraceOfSessionCheckout;
+  private final ActionOnInactiveTransaction actionOnInactiveTransaction;
```

Comment:
Yes, that can be done. But I felt it was a better semantic to have all our session leak options at one place. For ex - `ActionOnSessionLeak` and `ActionOnSessionNotFound` are just above this. `InactiveTransactionRemovalOptions` ideally wouldn't exist apart from being used in the unit tests to tweak other-wise private properties. Hence, it will be nice to have this separation between the two properties - `ActionOnInactiveTransaction` and `InactiveTransactionRemovalOptions` .

Reply:
I would follow the logic of `ActionOnInactiveTransaction` is part of the `InactiveTransactionRemovalOptions` (put another way: It is always used in combination with those settings). The fact that the other fields of `InactiveTransactionRemovalOptions` are (currently) package-private does not change that.